lib/constants/index.ts

```diff
@@ -278,8 +278,10 @@ export const offsetCompatibility: CompatMap = {
   fiat: ["bct", "nct", "mco2", "ubo", "nbo"],
 };
 
+// TODO - remove once env variable is added...
```

Comment:
still a TODO? env should be added.

But note that I am pushing for us to never use env vars in `lib` as it should be environment agnostic...

Reply:
@Atmosfearful I originally had the subgraph urls in here and must have left this comment here. I've actually moved the new env variables and subgraph url for bridged carbon to `app/lib/constants.ts` as part of this PR as I remembered you saying we shouldn't be adding env variables to `lib`. Will remove this comment.